code/modules/mob/emote.dm

```diff
@@ -1,3 +1,21 @@
+#define EMOTE_COOLDOWN 30		//Time in deciseconds that the cooldown lasts
+
+//Emote Cooldown System
+/mob/proc/handle_emote_CD(cooldown = EMOTE_COOLDOWN)
+	if(emote_cd == 3) //Spam those emotes
```

Comment:
no magic values, use one var to track time, if we need to do this we don't need to have an admin toggle
if you want an admin toggle (honestly would be a good idea imo) make it a config (very easy, define a /datum/config_entry/number/sound_emote_cooldown, and do CONFIG_GET(number/sound_emote_cooldown)